### ISSUE TITLE
There are some problems in MUMT.py

### ISSUE THREAD

**Issue by drqywang** [revenol/DDLO#4]
Thanks for your contribution, this helped me a lot. And when I read the code, I have the following confusions.
In MUMT.py  
what is the meaning of APP?
what is the meaning of ‘cu’? Is it the cost of uploading? What does 100/(8 *self.N) stand for?
Here you define CUL, but it is not used in the entire program.
Looking forward to your reply. Thanks

**Comment by revenol** [comment]
Thanks for your comments. We have updated MUMT.py and included more comments. To answer your questions:

> what is the meaning of APP?

A: The number of CPU cycles per bit required to complete a task/application, which depends on specific applications. In fact, some tasks/applications may require more CPU cycles. Although 'self.APP' is a fixed parameter here, the program is compatible with various applications.

> what is the meaning of ‘cu’? Is it the cost of uploading? 

A: 'self.cu' is the uploading speed.

> What does 100/(8 *self.N) stand for? Here you define CUL, but it is not used in the entire program.

A: Here the '100' is the value of 'self.CUL'. We have updated the code.
